cargo-dist/src/lib.rs

```diff
@@ -50,6 +50,7 @@ pub mod linkage;
 pub mod manifest;
 pub mod net;
 pub mod platform;
+pub mod platforms;
```

Comment:
I thought the same but eventually decided to go for that instead of having a `platform.rs` that's twice as long (and has a "mostly constants" section and a "mostly logic" section).

We should feel free to restructure this as needed.

Reply:
Just bringing up an off-issue discussion: we've decided to make this a `targets` mod inside `platform`.

Reply:
I _think_ ashley said to then move `platform.rs` to `platform/mod.rs` (since we now have `platform/targets.rs`) but I _think_ you said the way it was now was fine? So I'm not sure what to do and I'd vote we merge first and think up a policy later, probably.

For reference, both of these work:

```rust
crate/
  platform.rs
  platform/targets.rs
```

```rust
crate/
   platform/mod.rs
   platform/targets.rs
```

My preference is for the former because otherwise my code editor is full of `mod.rs` tabs and that's not super helpful. Some folks prefer the latter because, well, files are grouped into folders, so the file tree looks tidier I guess.

Reply:
so the codebase uses the mod.rs pattern. i don't feel strongly beyond that we should try to be consistent. i am fine with this being the first pass and we can slowly just update the others because i agree 100 tabs of "mod.rs" is not great